<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>card_id</class>
 <widget class="QWidget" name="card_id">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="card_num">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>330</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delet">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>330</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>411</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="search">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>卡号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>等级</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rank">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>手机号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phone_number">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="rank_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>270</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="pay_mem">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>余额</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
